docs/releases/v2019.1.rst

```diff
@@ -0,0 +1,229 @@
+
+To allow a migration from B.A.T.M.A.N. compat 14, this Gluon release offers the ability to ship both B.A.T.M.A.N adv compat versions 14 and 15 in the same image. The ``mesh.batman_adv.routing_algo`` option decides which module gets loaded and the scheduled domain switching functionality can be used to migrate between the two versions.
+
+See the :ref:`mesh <user-site-mesh>` section for the *site.conf* configuration of this feature.
+
+Outdoor Mode
+============
+
+Radio devices hosted outdoors are often affected by different regulation than if they were installed indoors. The outdoor mode allows for the reconfiguration of 5 GHz radios onto different channels and channel ranges. Honoring stipulations like DFS currently make it difficult to use the 5 GHz band for mesh networking because it's never certain that nodes will stay on the same channel.
+If enabled, by setting ``wifi5.outdoor_chanlist``, a number of devices that are commonly installed outdoors will have outdoor mode automatically enabled during their initial setup, specifically:
+
+* Ubiquiti
+
+    - Bullet M
+    - Litebeam M5
+    - Nanostation M5
+    - Nanostation M5 Loco
+    - Rocket M5
+    - Rocket M5 TI
+    - Unifi AC Mesh
+    - Unifi AC Mesh Pro
+    - Unifi Outdoor
+
+* TP-Link
+
+    - CPE510
+    - WBS510
+
+See the :ref:`wifi5 <user-site-wifi5>` section for the *site.conf* configuration of this feature.
+
+Device Deprecation
+==================
+
+The ar71xx-tiny and several devices in the ramips-rt305x target have been marked as deprecated. The `GLUON_DEPRECATED` flag was introduced to offer downstream communities the choice on how to deal with the ending support for those devices. Devices or targets marked as deprecated will very likely not be included in following Gluon releases anymore, usually due to their insufficient flash size.
+
+See the :ref:`Build configuration <user-site-build-configuration>` section for details.
+
+Hoodselector: Geolocation Mode
+==============================
+
+Hoodselector is a new concept for continuous reevaluation of the node's domain. As a first step we support its geolocation feature. It can be used to automatically switch nodes during their runtime between domains.
+
+See the :doc:`../package/gluon-hoodselector` documentation for details.
+
+
+x86 images support firstboot
+============================
+
+x86 images are now using squashfs instead of ext4 and can now have their configuration reset by using ``firstboot``.
+
+
+Bugfixes
+********
+
+* Fixes passwordless SSH access when gluon-authorized-keys was used without gluon-setup-mode. (`#1777 <https://github.com/freifunk-gluon/gluon/issues/1777>`_)
+
+- Fixes cross-domain leakage of respondd data by not joining the link-local multicast group on br-client. Nodes now only listen on ``[ff05::2:1001]:1001`` for respondd queries on br-client. This may require an update to your respondd querying tooling. (`#1701 <https://github.com/freifunk-gluon/gluon/issues/1701>`_)
```

Comment:
These two enumeration items use different bullet point characters

Reply:
Fixed in the whole document.